lib/event.c

```diff
@@ -544,7 +544,8 @@ static void initializer(void)
 }
 
 #define STUPIDLY_LARGE_FD_SIZE 100000
-struct event_loop *event_master_create(const char *name)
+#define FD_LIMIT_SIZE_FOR_VTYSH 1000
```

Comment:
so "VTYSH" is a little off-target, since there are plenty (dozen?) of non-vtysh places where there can be an issue

Reply:
fixed